examples/completion_models/app.py

```diff
@@ -39,37 +40,26 @@
 
 
 def call_llm(model, temperature, prompt, **kwargs):
-    if model in CHAT_LLM_GPT:
-        prompt = prompts["chat"]["input_prompt"].format(text=kwargs["text"])
-        openai.api_key = os.environ.get("OPENAI_API_KEY")
-        chat_completion = openai.ChatCompletion.create(
-            model="gpt-3.5-turbo",
-            messages=[{"role": "user", "content": prompt}],
-            temperature=0.7,  # Controls the creativity of the generated response
-            max_tokens=kwargs[
-                "maximum_length"
-            ],  # Controls the maximum length of the generated response
-            n=1,  # How many completions to generate
-            stop=kwargs["stop_sequence"],
-            top_p=kwargs["top_p"],
-            frequency_penalty=kwargs["frequence_penalty"],
-            presence_penalty=kwargs["presence_penalty"],
-        )
-        result = chat_completion.choices[0].message.content
-        return result
-    # replicate
+    selected_model = "gpt-3.5-turbo" # default to gpt-3.5-turbo
```

Comment:
I see here that you defaulted to gpt-3.5-turbo. Is there a way to use in addition the one specified in the CHAT_LLM_GPT variable

    "gpt-3.5-turbo-16k-0613",
    "gpt-3.5-turbo-16k",
    "gpt-3.5-turbo-0613",
    "gpt-3.5-turbo-0301",
    "gpt-3.5-turbo",
    "gpt-4",

?